contents/rust/Rust.md

```diff
@@ -0,0 +1,80 @@
+# Rust
+
+Author(s): [Tan Li Hao](https://github.com/LiHaoTan)
+
+## Why Rust
+Rust is a relatively new programming language, so as such the benefits of learning or using the language is not very clear. It is also important to know the merits of a language so we know when to use them.
```

Comment:
Might to good to mention that Rust supports both imperative and functional programming paradigm.